deploy/stacks/backend_stack.py

```diff
@@ -133,11 +132,15 @@ def __init__(
         )
 
         # Create the SES Stack
-        ses_stack = self.create_ses_stack(custom_domain, envname, kwargs, resource_prefix, email_notification_sender_email_id)
+        ses_stack = self.create_ses_stack(custom_domain, envname, kwargs, resource_prefix)
 
         repo = ecr.Repository.from_repository_arn(
             self, 'ECRREPO', repository_arn=ecr_repository
         )
+        if None not in [custom_domain, ses_stack]:
+            email_sender = custom_domain.get('email_notification_sender_email_id', None) if custom_domain.get('email_notification_sender_email_id', None) != None else f'noreply@{custom_domain.get("hosted_zone_name")}'
```

Comment:
So here we are saying that we are keeping the `email_notification_sender_email_id` is going to be in `custom_domain` of `cdk.json`. If we are going forward with this, then I will make changes in the documentation on `gh-pages` github branch. I was not thinking of having it in the `custom_domain` but I get it and that it makes sense to club these

Reply:
@TejasRGitHub, I think we are going to keep it as part of the custom_domain section. Because even if it is not a route53 parameter per se, it is not a standalone parameter. It can only exist if custom domain exists